e2e/actions/l2_challenger_test.go

```diff
@@ -59,7 +64,7 @@ func TestChallenger(gt *testing.T) {
 	// In fact, the following code is designed to create one or more finalized L2 blocks
 	// in order to pass the test after the Blue hard fork.
 	// If Proto Dank Sharding is introduced, the below code fix may no longer be necessary.
-	for i := 0; i < 2; i++ {
+	for i := 0; i < 3; i++ {
```

Comment:
Why it should be 3?

Reply:
Because the target block number of invalid output is set to 17.
For challenge actions test, submission interval is 6, thus it iterates 3 times to make invalid output submitted.